src/tiff.rs

```diff
@@ -759,3 +796,58 @@ pub fn read_exif_section(data: &[u8]) -> Result<Tiff, TiffError> {
 
     Ok(read_tiff(&mut cursor)?)
 }
+
+
+pub fn read_tiff_file(data: &[u8]) -> Result<Tiff, TiffError> {
```

Comment:
I hesitate to add this without adding similar functions for HEIC & JPEG, but if you don't feel like updating this I am fine as this would probably not the quality of this library further.

Reply:
I've made it `pub(crate)` for now.